cmd/replay/main.go

```diff
@@ -152,6 +160,8 @@ func main() {
 	db := state.NewDatabase(raw)
 
 	lastBlockHash := wavmio.GetLastBlockHash()
+	lastHotShotHeaderHash := wavmio.GetLastHotShotHeaderHash()
+	lastHotShotHeader := getHotShotHeaderByHash(lastHotShotHeaderHash)
```

Comment:
What do we gain from going via the last hash? Could we have a function `getHotShotHeader()` instead?

Reply:
Hmm. just noticed the types in `wavmio.stub` are all primitive, thinking probably for a reason.

Reply:
Yeah there are some wasm compilation quirks here I’m still figuring out